src/graphtypes/simplegraphs/simpleedgeiter.jl

```diff
@@ -82,5 +82,8 @@ function ==(e1::SimpleEdgeIter, e2::SimpleEdgeIter)
     return true   
 end
 
+in(e::SimpleEdge, es::SimpleEdgeIter) = has_edge(es.g, e)
+in(e::Union{Pair, NTuple{2}}, es::SimpleEdgeIter) = has_edge(es.g, e)
+
```

Comment:
Can this be one definition with a 3 part union?
in(e::Union{Pair, NTuple{2}, SimpleEdge}, es::SimpleEdgeIter) = has_edge(es.g, e)

Reply:
Yes, this only indicates that `in(e::SimpleEdge, es::SimpleEdgeIter) = has_edge(es.g, e)` is th only way to do it; whereas for the second line there are several options - like
`in(e::Union{Pair, NTuple{2}}, es::SimpleEdgeIter) = has_edge(es.g, first(e), last(e))` etc. and the chosen one happens to work nicely, but that was not clear.

Reply:
why restrict `e` at all?

`in(e, es::SimpleEdgeIter) = has_edge(es.g, e)`

Reply:
I was just conservative, we can do that.

Reply:
woah, Seth advocating for untyped function arguments, I am so proud

Reply:
Let this be a lesson to you youngsters: sometimes old dogs CAN learn new tricks.